package-lock.json

```diff
@@ -41762,6 +41763,17 @@
         "balanced-match": "^1.0.0"
       }
     },
+    "node_modules/typedoc/node_modules/marked": {
+      "version": "4.3.0",
+      "resolved": "https://registry.npmjs.org/marked/-/marked-4.3.0.tgz",
+      "integrity": "sha512-PRsaiG84bK+AMvxziE/lCFss8juXjNaWzVbN5tXAm4XjeaS9NAHhop+PjQxz2A9h8Q4M/xGmzP8vqNwy6JeK0A==",
+      "bin": {
+        "marked": "bin/marked.js"
+      },
+      "engines": {
+        "node": ">= 12"
+      }
+    },
```

Comment:
The 'marked' package version 4.3.0 is still being used in the 'typedoc' package. If the 'typedoc' package is not compatible with 'marked' version 9.0.0, this could potentially cause conflicts or unexpected behavior.